webcomponents/input/input.webcomponent.spec.js

```diff
@@ -133,6 +140,29 @@ describe('mn-input (webcomponent)', () => {
       expect(formID.test2).to.be.equal(element)
     })
   })
+
+  describe('attribute placeholder', () => {
+    it('should define a label to the placeholder', () => {
+      element.setAttribute('placeholder','test')
+      expect(element).to.contain('label').with.text('test')
+    })
+
+    it('should change the placeholder', () => {
+      element.setAttribute('placeholder','test')
+      element.setAttribute('placeholder','test2')
+      expect(element).to.contain('label').with.text('test2')
+    })
+
+    it('should verify the default value when placeholder is not setted', () => {
+      expect(element).to.contain('label').with.text('')
+    })
+
+    it('should set property value when attribute is setted and after removed', () => {
```

Comment:
maybe be better write this it like "should empty label when placeholder is removed"

the word "value" looks a little confuse